dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -2875,7 +2883,7 @@ public Node visitCompoundAssignment(CompoundAssignmentTree tree, Void p) {
                                         withoutAssignment(kind),
                                         tree.getVariable(),
                                         tree.getExpression());
-                        handleArtificialTree(operTree);
+                        handleArtificialTreeWithPath(operTree);
```

Comment:
What is the logic for changing these invocations and not others? There are still 32 calls of plain `handleArtificialTree` and I'm not sure I see the reason why some now have a Path and others don't.